T/taskwarrior/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+using BinaryBuilder
+using Pkg
+
+name = "taskwarrior"
+version = v"3.1.0"
+
+sources = [
+    GitSource("https://github.com/GothenburgBitFactory/taskwarrior", "5c6cc3e5229676655e93264e24a146a26b980e45")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/taskwarrior/
+
+mkdir build
+cd build
+
+cmake .. -B build -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=$prefix
```

Comment:
```suggestion
cmake .. -B build \
    -DCMAKE_BUILD_TYPE=RelWithDebInfo \
    -DCMAKE_INSTALL_PREFIX=$prefix \
    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
```
We need to include the toolchain to make sure the compilers are detected correctly.